README.md

```diff
@@ -193,6 +193,16 @@ More resources can be found:
 Examples for deployment via systemd and other use cases can be found in the [deployment
 repository](https://github.com/coredns/deployment).
 
+## Deprecation Policy
+
+When there is a backwards incompatible change in CoreDNS the following process is followed:
+
+* Release x.y.z: Announce that in the next release we will make backward incompatible changes.
+* Release x.y+1.0: Make the changes, but allow the old configuration to be parsed. I.e. CoreDNS will
```

Comment:
Instead of `x.y+n.n` notation,  maybe instead we can define that x.y.z, is major.minor.patch, then clearly state when major, and minor versions are incremented at each stage.